app/resources/profile_link_resource.rb

```diff
@@ -0,0 +1,7 @@
+class ProfileLinkResource < BaseResource
+  immutable
```

Comment:
oh... I mis-thought that. I forgot that someone could add a profile link and also remove it if they don't want it showing anymore... ops.